kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java

```diff
@@ -423,41 +422,29 @@ public void deactivateSubscription(Subscription subscription, DeactivationReason
     }
 
     @Override
+    @Cacheable("pack-type")
```

Comment:
Is it possible to combine the three subscription pack caches into one?  It probably isn't a big deal memory wise, but if someone were to copy this pattern for a entity with more objects then that could cause memory pressure.  

Either that or if in the cache config we set these to be small caches (like 2 objects each then we should be fine)